CHANGELOG.md

```diff
@@ -3,17 +3,36 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
-- Add `plotly[express]` extra for easily installing Plotly Express dependencies [#4644](https://github.com/plotly/plotly.py/pull/4644)
+- Add `plotly[express]` extra for easily installing Plotly Express dependencies [[#4644](https://github.com/plotly/plotly.py/pull/4644)]
+- Add `subtitle` attribute to all Plotly Express traces [[#4830](https://github.com/plotly/plotly.py/pull/4830)].
 
 ### Removed
-- Drop deprecated `pointcloud` and `heatmapgl` traces from the API
-- Drop `tenacity` dependency [#4831](https://github.com/plotly/plotly.js/pull/4831)
-
-### Updated
-
-- Updated plotly.py to use base64 encoding of arrays in plotly JSON to improve performance.
-- Add `subtitle` attribute to all Plotly Express traces
-- Make plotly-express dataframe agnostic via Narwhals [#4790](https://github.com/plotly/plotly.py/pull/4790)
+- Drop deprecated `pointcloud` and `heatmapgl` traces from the API [[#4815](https://github.com/plotly/plotly.py/pull/4815)]
+- Drop `tenacity` dependency [[#4831](https://github.com/plotly/plotly.py/pull/4831)]
+- Drop support for Jupyter Notebook version 6 and earlier [[#4822](https://github.com/plotly/plotly.py/pull/4822)]
```

Comment:
Which lower versions are supported?

Reply:
Yes for For Jupyter notebooks. 
Should you revise the following note?
```
Drop support for Jupyter Notebook version 6 and earlier
```

Reply:
You mean to specify the versions that do work as well as those that are dropped?

Reply:
Updated